CMakeLists.txt

```diff
@@ -57,13 +57,19 @@ elseif (APPLE)
     set(VULKAN_PREBUILT_BIN_GLSLC sdk/macos/bin/glslc)
     set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/macos/bin/spirv-cross)
 elseif (WIN32 AND NOT WINDOWS_STORE)
-    # Fallback to building for MacOS
     set(VULKAN_PREBUILT_HEADERS sdk/windows/Include/)
     set(VULKAN_PREBUILT_LIBRARY_PATH sdk/windows/Lib)
     set(VULKAN_PREBUILT_LIBRARY_NAME vulkan-1.lib)
     set(VULKAN_PREBUILT_DEPENDENT_LIBRARIES)
     set(VULKAN_PREBUILT_BIN_GLSLC sdk/windows/Bin/glslc.exe)
     set(VULKAN_PREBUILT_BIN_SPIRV_CROSS sdk/windows/Bin/spirv-cross.exe)
+elseif (UNIX)
```

Comment:
that's probably not specific enough (e.g. webos would still 'count' as UNIX) but it'll do for now (until such time as we add other linux-based platforms)